boards/arm/npcx7m6fb_evb/Kconfig.board

```diff
@@ -0,0 +1,6 @@
+# Copyright (c) 2020 Nuvoton Technology Corporation.
+# SPDX-License-Identifier: Apache-2.0
+
+config BOARD_NPCX7M6FB_EVB
+	bool "Nuvoton NPCX7M6FB EVB Development board"
+	depends on SOC_NPCX7M6FB
```

Comment:
help 

brief description of board and point to README

Reply:
I will add brief description of board later.